groups/assemble/assemble_unsigned_psbt.js

```diff
@@ -114,16 +120,40 @@ module.exports = ({capacity, proposed, rate}, cbk) => {
 
           const funded = sumOf(member.utxos.map(n => n.witness_utxo.tokens));
 
-          return [
-            {
-              script: member.funding,
-              tokens: capacity,
-            },
-            {
-              script: member.change,
-              tokens: funded - committed(capacity, proposed) - (vbytes * rate),
-            },
-          ];
+          const fee = vbytes * rate;
+
+          // Calculate total output amount (including funding outputs)
+          // Group channel opens have only one funding output
+          // Fanouts have multiple funding outputs
+          const calculateTotalOutputAmount = (member, capacity, proposed) => {
+            const memberCount = !!member.funding ? member.funding.length : 0;
```

Comment:
when is there zero members? is this for like a pair group? it seems confusing to have zero members

Reply:
Yes, there is a case where if it's a pair group, member.funding is undefined.